book/layout.md

```diff
@@ -246,11 +232,12 @@ create child layout objects:
 ``` {.python indent=4}
 def layout(self):
     if self.has_block_children():
+        previous = None
         for child in self.node.children:
-            if isinstance(child, TextNode): continue
```

Comment:
Not skipping TextNode any more?

Reply:
There no won't be whitespace-only TextNodes, which is what this was skipping